src/v/raft/tests/type_serialization_tests.cc

```diff
@@ -482,7 +482,7 @@ SEASTAR_THREAD_TEST_CASE(snapshot_metadata_roundtrip) {
       .cluster_time = ct,
       .log_start_delta = delta};
 
-    auto d = serialize_roundtrip_rpc(std::move(metadata));
+    auto d = serialize_roundtrip_rpc_adl(std::move(metadata));
```

Comment:
is this an example of one of the rpc messages that remain as adl (only?) ?

Reply:
The `snapshot_metadata` is used in the on-disk format.  I'll check if I'm using this method anywhere else and if not take the misleading 'rpc' out of the name

Reply:
oh right yeh that makes sense